CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.10
+
+- Added support for importing repos into a subsection.
+- Fixed moving the imported docs folder up so that the docs folder does not appear in the menu.
+
```

Comment:
Can you add this bullet? Feel free to edit it.

```markdown
- MkDocs versions  >= 1.4.0 changed the default config values from an empty string to None. You should versions >= to insure support with newer versions of MkDocs.
```

Reply:
@jdoiro3 I added the bullet point. I changed the second sentence, does that reflect what you meant?